app/src/main/java/com/example/mycleaninglog/ExpandableCard.kt

```diff
@@ -46,20 +46,20 @@ fun ExpandableCard(
     myCleaningTasks: List<cleaningTask>,
     viewModel: MainViewModel
     ) {
-    //Used for Expandable card level 1 functionality
+    /** Used for Expandable card level 1 functionality */
```

Comment:
Leave it as it were.  Inline comments should be //, especially if they are single line.  KDoc/JavaDoc is /*, and belongs above a class/method.